cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -702,6 +701,11 @@
 (type AMO (primitive AMO))
 (type VecMachLabel extern (enum))
 
+(type Signedness
+  (enum
+    (Signed)
+    (Unsigned)))
```

Comment:
If you decide to reuse `SelectReg` instead of adding a new pseudo-instruction then I suppose this question doesn't matter. Otherwise, feel free to leave the `Signedness` enum as-is.